src/components/Integrations/DataStudio/FreeAccount/FreeAccount.tsx

```diff
@@ -0,0 +1,63 @@
+
+import React, { Fragment } from 'react';
+import './styles/index.css'
+
+
+interface IFreeAccountProps {
+
+}
+
+const FreeAccount: React.SFC<IFreeAccountProps> = (props) => {
```

Comment:
Please note that React.SFC is deprecated from version 16.7 onwards, we are using react 17.0.5 right now https://github.com/FromDoppler/doppler-webapp/blob/master/package.json#L15
You should change `React.SFC` for `React.FunctionComponent`

Reply:
suggets acepted and applied